fe/fe-core/src/main/java/com/starrocks/load/routineload/RLTaskTxnCommitAttachment.java

```diff
@@ -64,11 +64,17 @@ public class RLTaskTxnCommitAttachment extends TxnCommitAttachment {
     private RoutineLoadProgress progress;
     private String errorLogUrl;
     private long loadedBytes;
+    private long timestampMs;
```

Comment:
If this timestampMs is in the TTxnCommitAttachment, it is not reasonable that this is stored in this class.
It should be in TxnCommitAttachment class.